sdk/elixir/getting_started.livemd

```diff
@@ -41,10 +41,10 @@ In this section, we will pull Elixir image from [hex.pm](https://hub.docker.com/
 After execute the code below, the library will printing out log from Dagger session to the standard output.
 
 ```elixir
-client
-|> Dagger.Query.container()
-|> Dagger.Container.from("hexpm/elixir:1.14.4-erlang-25.3-debian-buster-20230227-slim")
-|> Dagger.Container.with_exec(["elixir", "--version"])
-|> Dagger.Container.stdout()
-|> IO.puts()
```

Comment:
Is removing `IO.puts` mean nothing will be printed out in the terminal? 🤔

Reply:
By default `stdout` will print through dagger log.  So I think it is not necessary to add `IO.puts` in this cell. :)

Reply:
Here is demo 
<img width="978" alt="Screenshot 2566-08-04 at 21 03 30" src="https://github.com/dagger/dagger/assets/484530/f5c7cf75-a166-4010-80ae-e35da2e270bf">

Reply:
@wingyplus ah, that's actually nice, I didn't know that!